### PR TITLE
Access-control: continue TCs with other Pods when one fails.

### PR DESCRIPTION
**Pull request by greyerof**
+ Improve error logging to claim file.

TCs updated:
+ Pod's service account.
+ Pod's role binding with other namespaces.
+ Pod's cluster role bindings.

### CODE REVIEW

File: test-network-function/accesscontrol/suite.go [hunk 1]
```diff
@@ -54,6 +54,12 @@ var (
 	}
 )
 
+// failedPod is a helper type to track pods that fail in each of the TCs.
+type failedPod struct {
```
Comment (hamadise) [inline test-network-function/accesscontrol/suite.go:58]: why not storing pods directly ? then extracting  name and namespace later when needed ?

File: test-network-function/accesscontrol/suite.go [hunk 2]
```diff
@@ -308,10 +314,18 @@ func testServiceAccount(env *config.TestEnvironment) {
 	testID := identifiers.XformToGinkgoItIdentifier(identifiers.TestPodServiceAccountBestPracticesIdentifier)
 	ginkgo.It(testID, func() {
 		ginkgo.By("Should have a valid ServiceAccount name")
+		failedPods := []failedPod{}
 		for _, podUnderTest := range env.PodsUnderTest {
-			ginkgo.By(fmt.Sprintf("Testing pod service account %s %s", podUnderTest.Namespace, podUnderTest.Name))
+			ginkgo.By(fmt.Sprintf("Testing service account for pod %s (ns: %s)", podUnderTest.Name, podUnderTest.Namespace))
 			serviceAccountName := podUnderTest.ServiceAccount
-			gomega.Expect(serviceAccountName).ToNot(gomega.BeEmpty())
+			if serviceAccountName == "" {
```
Comment (hamadise) [inline test-network-function/accesscontrol/suite.go:321]: if serviceAccountName -> if podUnderTest.ServiceAccount

Comment (greyerof) [reply test-network-function/accesscontrol/suite.go]: Done.

File: test-network-function/accesscontrol/suite.go (hunk 1 above)
Comment (greyerof) [reply test-network-function/accesscontrol/suite.go]: You're right. Done.

**Review by hamadise — APPROVED** [review]

**Pull request merged**